artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/Mappers/BurnDown/RolePointsMapper.cs

```diff
@@ -0,0 +1,52 @@
+using Artificial.Scrum.Master.ScrumIntegration.Features.Burndown;
+using System.Text.RegularExpressions;
+
+namespace Artificial.Scrum.Master.ScrumIntegration.Mappers.BurnDown;
+
+internal interface IRolePointsMapper
+{
+    List<KeyValuePair<string, decimal>> MapTotalRolePoints(TotalPoints? totalPoints, SprintData sprintData);
+}
+
+internal partial class RolePointsMapper : IRolePointsMapper
+{
+    [GeneratedRegex(@"\D")]
+    private static partial Regex ExtractNumericValue();
```

Comment:
Przepraszam co to za czary xD. W sensie po co ten partial itd? jak dla mnie mogłeś tego regexa nazwać _numericValue czy coś zamiast robić takie kolcki. + ekstrakcja do osobnej klasy była bardziej dla celu testu jednostkowego którego i tak nie napisałeś

Reply:
ale już dobra ale wiesz, kiedyś będziesz musiał się wreszcie unitów nauczyć

Reply:
Visual Studio mi powiedział żebym tak zrobił lepiej bo wtedy on sobie to zrobi w Compile Time i jest niby bardziej wydajne czy coś